portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,7 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.img {
+    width: 100%;
```

Comment:
Please use consistent indentation, (either 2-space or 4-space indentation but make sure it's uniform across this file)